lms/djangoapps/courseware/views.py

```diff
@@ -815,6 +815,7 @@ def course_about(request, course_id):
                 reg_url=reverse('register_user'), course_id=course.id.to_deprecated_string())
 
         course_price = get_cosmetic_display_price(course, registration_price)
+        is_shopping_cart_enabled = _is_shopping_cart_enabled and registration_price
```

Comment:
nitpick on the name here, since you have basically just redefined is_shopping_cart_enabled. Can I counter-propose: has_shopprting_cart_price?